m2cgen/interpreters/code_generator.py

```diff
@@ -16,6 +16,8 @@ def __call__(self, *args, **kwargs):
 
 class BaseCodeGenerator:
 
+    initial_code = ""
```

Comment:
I don't think this code deserves to be in a BaseGenerator, considering that it's way to language-specific + we do initial generation in the top level `interpret` method of the corresponding interpreter. Also I'm not sure I understand why `assign_array` needs to be "initial" since we need to add it only in case of classification.

Reply:
Agreed, fixed.